### PR TITLE
Migrate staging/src/k8s.io/cloud-provider/plugins.go to structured logs

### PR DESCRIPTION
**Pull request by CriaHu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
[migration-to-structured-logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
Migrate some scheduler log messages to structured logging
```

### CODE REVIEW

**Comment by shubheksha** [comment]
/retitle Migrate staging/src/k8s.io/cloud-provider/plugins.go to structured logs

File: staging/src/k8s.io/cloud-provider/plugins.go [hunk 1]
```diff
@@ -117,16 +117,16 @@ func InitCloudProvider(name string, configFilePath string) (Interface, error) {
...
 		return nil, nil
 	}
 
 	if configFilePath != "" {
 		var config *os.File
 		config, err = os.Open(configFilePath)
 		if err != nil {
-			klog.Fatalf("Couldn't open cloud provider configuration %s: %#v",
-				configFilePath, err)
+			klog.ErrorS(err,"Couldn't open cloud provider configuration","configFilePath",
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/cloud-provider/plugins.go:128]: klog.ErrorS(err,"Couldn't open cloud provider configuration", "configFilePath",

**Comment by yangjunmyfm192085** [comment]
@CriaHu Are you still working on this pr?
Could you modify it?

**Pull request closed without merging**